content/docs/csidriver/installation/operator/_index.md

```diff
@@ -8,6 +8,17 @@ description: >
 
 {{% pageinfo color="primary" %}}
 The Dell CSI Operator is no longer actively maintained or supported. It will be deprecated in CSM 1.9. It is highly recommended that you use [CSM Operator](../../../deployment/csmoperator) going forward.
+
+Please follow the steps below to migrate from Dell CSI Operator to CSM Operator
+* Step1: Backup the CRD to save the settings used
+* Step2: Map and update the settings in CRD in step1 to the relevant CRD in CSM Operator
+* Step3: Keep the secret and namespace for the driver
+* Step4: Keep the Storage Class and Volume Snapshot Class
+* Step5: Uninstall the CRD from the CSI Operator
+* Step6: Uninstall the CSI Operator itself
+* Step7: Install the CSM Operator
+* Step8: Install the CRD updated in Step 2
+
```

Comment:
I thought we said we will have a table or a not that points to the related samples from each repo

![image](https://github.com/dell/csm-docs/assets/49235405/beae79ba-4b60-45a3-9363-172a9e1c0896)

Reply:
Sure @coulof.. We can get these changes in the follow-up PR. Sry this PR got merged before addressing this review comment